server/app/routes/oauth2_api.rb

```diff
@@ -24,7 +24,7 @@ def params_from_anywhere
        params =
          case request_content_type
          when JSON_MIME then JSON.parse(request.body.read)
-         when FORM_MIME then URI.decode_www_form(request.body.read)
+         when FORM_MIME then Hash[URI.decode_www_form(request.body.read)]
```

Comment:
[`URI.decode_www_form`](https://ruby-doc.org/stdlib-2.3.1/libdoc/uri/rdoc/URI.html#method-c-decode_www_form) returns `Array` of `[[key, value]]`